app/javascript/components/patient/assessment/actions/MonitoringPeriod.js

```diff
@@ -22,7 +22,7 @@ class MonitoringPeriod extends React.Component {
   render() {
     return (
       <Row>
-        <Col lg={8} sm={this.props.patient.isolation ? 12 : 8}>
+        <Col lg={this.props.patient.isolation ? 12 : 8} sm={this.props.patient.isolation ? 12 : 8}>
```

Comment:
these two breakpoints are redundant because they are the same now - just keep the small one

Reply:
unless you wanted to make the three columns in exposure stack on an xs screen or something, they're a bit cramped

Reply:
great catch, I knew something looked a little off about this: https://github.com/SaraAlert/SaraAlert/pull/950/files#diff-ad6312678a7202f7d64e70f935e165c195271aa1e6d8a4d390024a258f448764R25